### PR TITLE
News wizard: Skip the first type step except older services with older OSA loyalty

### LINKED ISSUE #880: Skipping the first news step
When sending news & actions, the first step should not be shown (always type 'news') UNLESS the service has the old 'OSA Loyalty' functionality (so not the 'OSA Terminal' functionality. Both use `SolutionModule.LOYALTY`, but when `SolutionModule.JOYN` is available, then our loyalty can only be used for OSA Terminal (tablet without scan function))

### PR DESCRIPTION
**Pull request by abom**
Set the default type to news always.

Closes #880.

### CODE REVIEW

File: src/static/js/solutions/sln-news-wizard.js [hunk 1]
```diff
@@ -1440,33 +1432,6 @@ NewsWizard.prototype = {
...
-                                joynUrl = 'https://www.joyn.be/en/for-merchants'
-                            }
-                            var message = T('joyn_enable_via_coupon').replace('%(joyn_url)s',
-                                    '<a href="' + joynUrl + '" target="_blank">' + joynUrl + '</a>');
-
-                            sln.alert(message, null, CommonTranslations.ERROR);
-                            return;
-                        }
-                    } else if (!MODULES.includes('loyalty')) {
-                        var message = T('contact_support_to_order_tablet_for_news_coupons');
```
Comment (RubenMattan) [inline src/static/js/solutions/sln-news-wizard.js]: Does this needs to be removed?

Comment (abom) [reply src/static/js/solutions/sln-news-wizard.js]: The issue state that the default type would be always TYPE_NEWS except for services with older loyalty.

Comment (RubenMattan) [reply src/static/js/solutions/sln-news-wizard.js]: What if loyalty is active and limited is false?

Comment (abom) [reply src/static/js/solutions/sln-news-wizard.js]: Then the step would appear normally, and he can create or edit news items.

**Pull request merged**